scripts/upgrade-ssv-network-views.ts

```diff
@@ -1,7 +1,7 @@
-const { ethers, upgrades } = require("hardhat");
+import { ethers, upgrades } from 'hardhat';
 
 async function upgradeSSVNetworkViews() {
-  const proxyAddress = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
+  const proxyAddress: any = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
```

Comment:
Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)